tools/pythonpkg/src/include/duckdb_python/vector_conversion.hpp

```diff
@@ -0,0 +1,55 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb_python/array_wrapper.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb_python/pybind_wrapper.hpp"
+
+#include "duckdb.hpp"
+
+namespace duckdb {
+
+enum class PandasType : uint8_t {
```

Comment:
Just a minor stylistic thing, but enum may as well be called NumpyType, which is a bit more generic :)